server/channels/app/platform/config.go

```diff
@@ -39,6 +39,8 @@ type ServiceConfig struct {
 // ensure the config wrapper implements `product.ConfigService`
 var _ product.ConfigService = (*PlatformService)(nil)
 
+var fetchUserCount = true
```

Comment:
0/5 if you want to rename this

Reply:
Maybe just be a bit more specific `fetchUserCountForFirstUserAccountCheck` but that's just a nit

Reply:
Maybe `shouldFetchUserCount`? Also would this make more sense as a field on `PlatformService` rather than a global?